llvm-general/test/LLVM/General/Test/Instrumentation.hs

```diff
@@ -144,13 +145,13 @@ tests = testGroup "Instrumentation" [
         (names ast') `List.intersect` (names ast) @?= names ast
     | (n,p) <- [
      ("GCOVProfiler", defaultGCOVProfiler),
-{-
-     ("AddressSanitizer", defaultAddressSanitizer),
-     ("AddressSanitizerModule", defaultAddressSanitizerModule),
--}
-     ("MemorySanitizer", defaultMemorySanitizer),
+     -- ("AddressSanitizer", defaultAddressSanitizer),
+     -- ("AddressSanitizerModule", defaultAddressSanitizerModule),
      ("ThreadSanitizer", defaultThreadSanitizer),
-     ("BoundsChecking", BoundsChecking)--,
-    ]
+     ("BoundsChecking", BoundsChecking)
+    ] ++ if Info.os == "linux" -- or FreeBSD
```

Comment:
I don't know that GHC will produce the string "FreeBSD" when run on FreeBSD, but LLVM itself makes this distinction.